app/src/main/java/org/mifos/mobile/ui/qr_code_display/QrCodeDisplayTopBar.kt

```diff
@@ -0,0 +1,33 @@
+package org.mifos.mobile.ui.qr_code_display
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Share
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import org.mifos.mobile.R
+import org.mifos.mobile.core.ui.component.MifosTopBar
+
+@Composable
+fun QrCodeDisplayTopBar(
```

Comment:
This should be generic type, Current generic type topbar in designsystem can be used here

Reply:
updated